core/src/env_writer.rs

```diff
@@ -338,13 +339,13 @@ mod tests {
 
     use crypto_bigint::U256;
 
-    use secp256k1::hashes::Hash;
-
     use crate::{
         env_writer::ENVWriter, errors::BridgeError, merkle::MerkleTree, mock::env::MockEnvironment,
         utils::parse_hex_to_btc_tx,
     };
 
+    use secp256k1::hashes::Hash;
```

Comment:
This test module has use statements all over the place. Not a fan of that tbh. Not a strict change request, though.